json-endpoints/class.wpcom-json-api-get-customcss.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Custom Css endpoint
+ *
+ * https://public-api.wordpress.com/rest/v1.1/sites/$site/customcss/
+ */
+
+class WPCOM_JSON_API_Get_CustomCss_Endpoint extends WPCOM_JSON_API_Endpoint {
+	/**
+	 * API callback.
+	 */
+	function callback( $path = '', $blog_id = 0 ) {
+		// Switch to the given blog.
+		$blog_id = $this->api->switch_to_blog_and_validate_user( $this->api->get_blog_id( $blog_id ) );
+		if ( is_wp_error( $blog_id ) ) {
+			return $blog_id;
+		}
+
+		$args = array(
+			'css' => Jetpack_Custom_CSS::get_css(),
```

Comment:
Just for the record, we discussed that yesterday with @sirbrillig and he said that it's OK to leave this endpoint out for now. So I'm going to rebase this branch and remove this commit.